File: Selenium/Activities/Activity3.java
package Activities;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.*;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {
    public static void main(String[] args) throws InterruptedException
    {
        WebDriverManager.firefoxdriver().setup();
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriver newDriver = new FirefoxDriver();
        newDriver.get("https://training-support.net/selenium/login-form");
        System.out.println("Title of page is: "+ newDriver.getTitle());
        newDriver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
        newDriver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");
        newDriver.findElement(By.xpath("//button[@type='submit']")).click();
        Thread.sleep(200);
        newDriver.quit();
    }
}

File: Selenium/Project/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("https://alchemy.hguy.co/lms")
first_info_box = driver.find_element(By.XPATH, "//h3[text()=' Actionable Training ']")
assert first_info_box.text == "Actionable Training"
driver.quit()